pkg/didmethod/peer/did.go

```diff
@@ -25,8 +25,23 @@ const (
 	peerPrefix = "did:peer:"
 )
 
-//newDid creates the peer DID. For example : did:peer:11-479cbc07c3f991725836a3aa2a581ca2029198aa420b9d99bc0e131d9f3e2cbe
-func newDid(doc *did.Doc) (string, error) {
+//NewDID returns the resolved variant of the genesis version of the peer DID document
+func NewDID(publicKey []did.PublicKey, authorization []did.VerificationMethod) (*did.Doc, error) {
```

Comment:
I feel a bit confused about this method name. I would expect NewDID to construct a DID object, but this is constructing a Doc.

Reply:
@troyronda what you suggest ? Basically this doc will have computed DID embedded inside the genesis doc. Does NewDoc resonates or  ResolvedDoc.  Kindly suggest

Reply:
I think a method with the prefix New should have the suffix marching the type it returns.

Reply:
@llorllale can you please move this to framework development board under peer review please

Reply:
Done